ci/test_commit_message.py

```diff
@@ -9,8 +9,18 @@
 import re
 import sys
 
+
 # This script is written to be used with CI Integration
 
+def hasUrl(string):
```

Comment:
Small nit: Please change this to `has_url`.

Reply:
Done